public/static/app/perfdiscovery/PerfDiscoveryDataService.js

```diff
@@ -237,6 +237,7 @@ mciModule.factory('PerfDiscoveryDataService', function (
           taskName: taskName,
           buildName: build.name,
           buildVariant: build.variant,
+          project: build.project,
```

Comment:
[nit] the returns comment is out of date. 

_.chain might simplify the mapping and remove the need for reduce (it won't make any less complex though).